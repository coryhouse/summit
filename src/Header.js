import React from "react";
import PropTypes from "prop-types";

const Header = ({ companyName = "Summit LLC" }) => {
  return (
    <div style={{ color: "white", backgroundColor: "#52B6E8" }}>
      <h1>{companyName}</h1>
    </div>
  );
};

Header.propTypes = {
  companyName: PropTypes.string
};

export default Header;
