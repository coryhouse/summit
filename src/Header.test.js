import React from "react";
import Header from "./Header";
import renderer from "react-test-renderer";

it("renders Summit in header by default", () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders the company name passed", () => {
  const tree = renderer.create(<Header companyName="Test company" />).toJSON();
  expect(tree).toMatchSnapshot();
});
