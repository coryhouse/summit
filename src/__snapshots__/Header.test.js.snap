// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Summit in header by default 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#52B6E8",
      "color": "white",
    }
  }
>
  <h1>
    Summit LLC
  </h1>
</div>
`;

exports[`renders the company name passed 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#52B6E8",
      "color": "white",
    }
  }
>
  <h1>
    Test company
  </h1>
</div>
`;
